coriolis/providers/backup_writers.py

```diff
@@ -8,6 +8,7 @@
 import threading
 
 from oslo_log import log as logging
+from oslo_config import cfg
```

Comment:
Nit: this should come before `oslo_log`,

Reply:
done